coralproject/talk
# [CORL-2389]: Show error message when trying to reply to rejected comment

## What does this PR do?

These changes add an error message that shows if someone already has a rejected comment loaded on their screen and tries to reply to it. This is for https://vmproduct.atlassian.net/browse/CORL-2389.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

As user 1, leave a comment on the stream. Load that comment up in the stream on another tab as user 2. Have user 3 reject the comment. Have user 2, who already has the comment loaded up, attempt to reply to the comment that was rejected by user 3. See that when user 2 attempts to submit their reply, they see the error message "The comment you are replying to has been rejected by a moderator."
 
## How do we deploy this PR?
